pkg/reconciler/v1alpha1/taskrun/resources/input_resource_test.go

```diff
@@ -522,7 +522,7 @@ func TestAddResourceToBuild(t *testing.T) {
 		wantErr: false,
 		want: buildv1alpha1.BuildSpec{
 			Steps: []corev1.Container{{
-				Name:  "create-dir-workspace-mz4c7",
+				Name:  "create-dir-workspace-0-0-mz4c7",
```

Comment:
why is this `0-0` added? container name should not have changed by this feature

Reply:
This is related to the following item of the commit message
```
Make sure `create-dir` containers gets a unique name (the same way `copy-` containers do)
```

Reply:
but they have a random string appended to them, why do we need the index?

Reply:
@pivotal-nader-ziada ah good point, that was done "before" we had that random string… :sweat_smile: We should remove that then :angel: